tests/unit/release/createPullRequest/release.spec.js

```diff
@@ -125,29 +133,32 @@ describe('src/release.js createPullRequest', () => {
     test('should set data.pr', async () => {
         expect.assertions(1);
 
-        const html_url = 'testUrl';
-        const apiUrl = 'apiUrl';
+        const html_url = 'apiUrl';
         const number = 42;
         const id = 'pr_id';
+        //const fullName = 'repoName';
         const expectedPR = {
-            url: html_url,
-            apiUrl,
-            number,
-            id
+            url: 'apiUrl',
+            apiUrl: 'apiUrl',
+            number: 42,
+            id: 'pr_id',
+            full_name: 'fullName'
         };
         const release = releaseFactory({ branchPrefix, releaseBranch });
         jest.spyOn(release, 'getMetadata').mockImplementationOnce(() => ({ repoName }));
         const createReleasePR = jest.fn(() => ({
             state: 'open',
-            html_url,
-            url: apiUrl,
+            html_url: html_url,
+            url: html_url,
             number,
-            id
+            id,
+            head: { repo: { full_name: 'fullName' } }
         }));
+        const addLabel = jest.fn();
         github.mockImplementationOnce(() => {
             //Mock the default export
             return {
-                createReleasePR
+                createReleasePR, addLabel
```

Comment:
Can you prettify the changed code such as this part?

Reply:
updated